### ISSUE TITLE
NotImplementedError

### ISSUE THREAD

**Issue by ryknehho** [F33RNI/GPT-Telegramus#44]
Hi! I apologize in advance if my questions are too noob. This looks like a wonderful project, but I'm having trouble getting it to run. I'm getting an error after starting it up, in the QueueHandler

Traceback (most recent call last):
  File "xxxx/GPT-Telegramus/QueueHandler.py", line 450, in _queue_processing_loop
    if self.request_response_queue.qsize() == 0:
  File "xxxx/.pyenv/versions/3.10.12/lib/python3.10/multiprocessing/queues.py", line 126, in qsize
    return self._maxsize - self._sem._semlock._get_value()
NotImplementedError

Am I missing something in the configuration file or missed installing some requirements? As you can see I'm running 3.10.12 installed using pyenv. I'm trying to run this on a Mac M1.

**Comment by F33RNI** [comment]
Please try to install the package versions specified in requirements.txt and try again

**Comment by F33RNI** [comment]
Perhaps you are using a newer version and the call in your error is really deprecated.  Unfortunately I won't be able to watch it soon (cause on the road now)
Thank you for opening this Issue, I'll take a look

**Comment by ryknehho** [comment]
Actually I just tried doing a fresh clone and pull on my Linux machine (it worked with python 3.10.2, and I also installed and tried with 3.10.12) instead of the Mac, and it runs beautifully, no errors. All configuration options are exactly the same. I'm still figuring out the configuration for ChatGPT but bard functionality is perfect.

It might just be some implementation issue on the Mac M1. 

Sorry for the false alarm!
